docs/debug.md

```diff
@@ -0,0 +1,62 @@
+### To debug this project in VS Code need to select python interpreter from virtual environment created by poetry.
```

Comment:
@SuelenKarbivnychyy 👋 

This should not be the heading as this is the first line of the debugging documentation. We should have a more concise heading

Reply:
Thank you, I will address it :)